includes/functions.php

```diff
@@ -459,21 +459,6 @@ function convertkit_select2_enqueue_styles() {
  */
 function convertkit_get_file_contents( $local_file ) {
 
-	// Call globals.
-	global $wp_filesystem;
-
-	// Load filesystem class.
-	require_once ABSPATH . 'wp-admin/includes/file.php';
-
-	// Initiate.
-	WP_Filesystem();
-
-	// Bail if the file doesn't exist.
-	if ( ! $wp_filesystem->exists( $local_file ) ) {
-		return '';
-	}
-
-	// Return file's contents.
-	return $wp_filesystem->get_contents( $local_file );
+	return file_get_contents( $local_file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

Comment:
Is it okay to return false if something goes wrong? Shouldn't this check that return an empty string? At the very least we need to update the type signature in the contents. I'm also kinda ambivalent about making this change since your assessment in the ticket seems like its a config issue restricted to one customer.

Reply:
Makes sense!